app/components/MeToken/createMetoken.tsx

```
@@ -77,41 +77,44 @@ export default function CreateMetoken() {
   const { data: isOwner, isPending: isPendingOwner } = useReadContract({
     contract: metokenDiamondBase,
     method: 'isOwner',
-    args: [activeAccount?.address],
+    params: [activeAccount?.address!],
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)